### ISSUE TITLE
Version 1.3 causes problem 

### ISSUE THREAD

**Issue by Superkloton** [nikonratm/feedlybackgroundtabs#7]
When directly opening the next article with the 'j' key (title only or magazine view), the latest version doesn’t open a tab at all. When assigning 'v' it opens a tab in the foreground instead.

I hope you can reproduce it. Go to any article and press 'j' to open the next article. Then press the assigned key (I use the 'i' key) to open the article in a background tab. Nothing happens or a foreground tab opens when 'v' is assigned.

Sebastian

**Comment by nikonratm** [comment]
Hi Sebastian, sorry I cannot reproduce this as the extension is working for me. If you can give more details on your setup, OS version, Safari version, what mode you're in on feedly, etc.

**Comment by Superkloton** [comment]
This is reproducible every single time for me. Of course I restarted Safari and the computer already.
macOS 10.12, Safari 10.0. Feedly "Title only" or "Magazine view", as I already mentioned. MacBook Air 2010 (C2D), if that makes any difference. Please let me know what else you need to know.

**Comment by nikonratm** [comment]
Please remove and re-install the extension. Finally, if its still not working, repeat the steps on your machine to reproduce the error and send me the contents of the error console (see http://ggnome.com/wiki/Using_The_Browser_Error_Console )

Thanks

**Comment by Superkloton** [comment]
I reinstalled the extension from Apple’s extension gallery to no avail. However, I deinstalled that version again and directly installed the version from GitHub and this works.
Since only version 1.3 is available in Apple’s extension gallery I guess the problem has been fixed in the latest version 1.4 that I use now.
Unfortunately, Apple is terribly slow with new extension versions.

Thanks for your help! Hope the latest version is available in the extension gallery soon.
